smoke/runtime/runtime_test.go

```diff
@@ -175,3 +183,23 @@ func allTrue(bools []bool) bool {
 	}
 	return true
 }
+
+func curl(skip bool, url string) (string, error) {
+	tr := &http.Transport{
```

Comment:
Do you mind calling this variable `transport` instead of `tr`?  We like the longer, more descriptive name.